test/DSROracleIntegrationGnosis.t.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "./DSROracleXChainIntegrationBase.t.sol";
+
+import { GnosisDomain } from "xchain-helpers/testing/GnosisDomain.sol";
+
+import { DSROracleForwarderGnosis }  from "../src/forwarders/DSROracleForwarderGnosis.sol";
+import { DSROracleReceiverGnosis } from "../src/receivers/DSROracleReceiverGnosis.sol";
+
+contract DSROracleIntegrationGnosisTest is DSROracleXChainIntegrationBaseTest {
+
+    DSROracleForwarderGnosis forwarder;
+    DSROracleReceiverGnosis receiver;
+
+    function setupDomain() internal override {
+        remote = new GnosisDomain(getChain('gnosis_chain'), mainnet);
+
+        mainnet.selectFork();
+
+        forwarder = new DSROracleForwarderGnosis(address(pot), computeCreateAddress(address(this), 5));
+
+        remote.selectFork();
+
+        oracle = new DSRAuthOracle();
+        receiver = new DSROracleReceiverGnosis(0x75Df5AF045d91108662D8080fD1FEFAd6aA0bb59, 1, address(forwarder), oracle);
```

Comment:
Can we make this address a constant? Not sure what its supposed to be

Reply:
done